models/app.models.js

```diff
@@ -14,4 +14,12 @@ exports.readAllEndpoints = () => {
     return fs.readFile("./endpoints.json", "utf-8").then(endpointsData => {
         return endpointsData
     })
+}
+
+exports.selectArticleById = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(article => {
+        if (article.rows.length === 0) return Promise.reject({status: 404, msg: "Not found"})
+        return article.rows
```

Comment:
If selecting hte first one, ensure you get `rows[0]` to only return what you need